docs/variants/qemu_q35/development.md

```diff
@@ -0,0 +1,138 @@
+# Dasharo (UEFI) v0.1.0 for QEMU Q35
+
+## Intro
+
+1. Please check your qemu setup and launch qemu outside
+docker instance (host machine) using built OVMF firmware image.
+
++ QEMU Installation:
+    - [Install QEMU on your linux distro](<https://www.qemu.org/download/#linux>)
+
+ ```bash
+ qemu-system-x86_64 -drive if=pflash,format=raw,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd -debugcon file:debug.log -global isa-debugcon.iobase=0x402 -global ICH9-LPC.disable_s3=1 -net none -machine q35,smm=on
+ ```
+
++ `-drive` indicate device is pflash with firmware image of built OVMF.fd image.
+
++ `-debugcon file:debug.log -global isa-debugcon.iobase=0x402`
+default OVMF build writes debug messages to IO port **0x402**.
+The following qemu command line options save them in the file called debug.log.
+
++ `-global ICH9-LPC.disable_s3=1` SMM is put to use in the S3 suspend
+and resume infrastructure, and in the UEFI variable driver stack.
+Similarly, a pflash-backed variable store is a requirement.
+
++ `-net none` disables the OptionROM execution which
+will interfere with the firmware image, if not disabled.
+
++ `-machine q35,smm=on` For SMM to work,
+only Q35 machines are supported hence the machine type.
+
+1. After executing the above qemu command,
+   qemu boots into UEFI built-in shell and to BIOS selection area.
+
+1. The features which are enabled in `OvmfPkgX64.dsc` can be verified at `BIOS menu`.
```

Comment:
`BIOS Menu`? I don't recall anything like that in EDK2 setup application. If anything, I would call it `Setup menu`